aten/src/ATen/ExpandUtils.cpp

```diff
@@ -25,8 +25,8 @@ Container infer_size_impl(IntArrayRef a, IntArrayRef b) {
     ptrdiff_t offset = ndim - 1 - i;
     ptrdiff_t dimA = dimsA - 1 - offset;
     ptrdiff_t dimB = dimsB - 1 - offset;
-    int64_t sizeA = (dimA >= 0) ? a[dimA] : 1;
-    int64_t sizeB = (dimB >= 0) ? b[dimB] : 1;
```

Comment:
Since these can be symInts now, you should call std::move when they are stored in the output below.

Reply:
Specifically:
```
      expandedSizes[i] = sizeA == 1 ? std::move(sizeB) : std::move(sizeA);
```